charts/stable/frigate/templates/configmap.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "common.names.fullname" . }}-config
+  labels:
+    {{- include "common.labels" . | nindent 4 }}
+data:
+  config.yml: |
+    {{- .Values.config | nindent 4 }}
```

Could be nice to allow templating here
```suggestion
    {{- .Values.config | tpl | nindent 4 }}
```

---

Good point! Added.